src/ComposerRequireChecker/NodeVisitor/DefinedSymbolCollector.php

```diff
@@ -45,6 +45,8 @@ public function enterNode(Node $node)
         $this->recordFunctionDefinition($node);
         $this->recordConstDefinition($node);
         $this->recordDefinedConstDefinition($node);
+
+        return $node;
```

Comment:
PHPStan was complaining about no return because of method's signature in PhpParser lib